src/index.html

```diff
@@ -5,10 +5,10 @@
   <title>Frontend</title>
   <base href="/">
 
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <meta content="width=device-width, initial-scale=1" name="viewport">
+  <link href="favicon.ico" rel="icon" type="image/x-icon">
 </head>
 <body>
-  <app-root></app-root>
+<app-root></app-root>
```

Comment:
to też nie ma znaczenia, ale ładne było to wcięcie ;c

Reply:
Ha, a to znalazłam